patches/patches.json

```diff
@@ -217,5 +217,13 @@
         "files": [
             "src/components/views/rooms/MemberList.tsx"
         ]
+    },
+    "add-feature-flag-based-on-homeserver": {
```

Comment:
`tchap-features` ? `tchap-features-from-config`?
(our patch names are generally super long, so I'm trying to reduce them)